src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -176,6 +178,9 @@ public static void InstallLocalCertificateFromBridge()
                 }
             }
 
+            // Set the local variable with the thumbprint so a test case can use it.
+            Thumbprint = thumbprint;
```

Comment:
Consider moving this further up where the PUT happens. We already know the cert thumbprint at that point.